templates/case-studies/index.html

```diff
@@ -1,72 +1,100 @@
+    {%- endif -%}
+    {%- if slot == 'highlights_item_content_4' -%}
+      <p>ESOC is responsible for a wide range of IT infrastructure and systems to help fly these unique missions, having been responsible for flying more than 80 custom-built satellites since it was established.</p>
+    {%- endif -%}
+  {%- endcall %}
 
   <!-- Text section -->
-  {% import "case-studies/templates/_macros-text.jinja" as text_template %}
-  {{ text_template.text(
-      title="You can’t explore space without infrastructure.",
-      description_list=[
-        "It’s not just expert personnel, satellites and spacecraft: it takes mountains of software, computer systems, data and networks to get these missions off the ground. These missions are often unique – testing the limits of our technological know-how, having never been done before with each mission presenting its own unique challenges. Building and deploying these systems on a mission-bymission basis often involves significant complexity and costs.",
-        "ESA decided to embark on an ambitious challenge: to fly double the number of satellites it does currently by 2030, looking to technology to help it achieve this goal, with more rapid and automated software delivery, shared services, more efficient use of resources and lower costs of deployment.",
-        "To meet this goal, ESA engaged Canonical to deliver the architecture, design and deployment of Kubernetes and Ceph for persistent storage. ESA also chose Canonical’s Ceph distribution to consolidate the various storage technologies in use. Canonical’s Kubernetes distribution allowed ESA’s Missions Operations Infrastructure – IT (MOI-IT) team to easily deploy its workloads, integrated with the existing IT infrastructure services, giving it a greater ability to support ESA missions and those of its partners, with automatic updates and deployment, 24/7 monitoring and reduced system complexity.",
-        "Kubernetes was perfect for ESA’s next generation multi-mission platform at ESOC. This open source system helps organisations orchestrate their workloads more efficiently with automation. However, Kubernetes’ fast-moving and steep learning curve makes it more demanding to manage. To reduce the engineering workload and management requirements of this new system, ESA chose Canonical to manage its Kubernetes deployments, providing updates, monitoring and maintenance.",
-        "The systems Canonical deployed and now manage for ESA mean that ESOC engineering teams can quickly and centrally manage and deploy mission-critical infrastructure and software, with more security, reliability, and control, and with less downtime and reduced risk of system failures."
-      ]
-    )
-  }}
+  {%- call(slot) text(title="You can’t explore space without infrastructure.") -%}
+    {%- if slot == 'paragraph_1' -%}
```

Comment:
Following up on our discussion in MM, I don't think these paragraphs really need to be separate slots in a range. Maybe you can make a single slot for paragraph content, and pass all of these in there?

My reasoning is that the paragraph-handling part of the text macro seems to be simple enough that the extra work of calling a macro isn't really worth it for these paragraphs, since the macro isn't adding or abstracting all that much.

Reply:
Thanks for the suggestion, I definitely agree that it is better to pass it in a single slot. The only issue is that ideally we'd like the content writers to use these macros, so if they were to pass the text chunks in a single slot they would have to manually define breakpoints/paragraphs (e.g using `br` or `\n`). Thus the separate slots were introduced to deal with paragraphing each text chunk.

Is there a better approach to do so if we want to eliminate the need of content writers to define breakpoints?

Reply:
> Is there a better approach to do so if we want to eliminate the need of content writers to define breakpoints?

Like we talked about in MM, while content writers should define breakpoints (with line breaks in the paragraphs they type), I don't see them doing it in HTML. 

I don't currently see a better solution for this, but if we absolutely had to support something like that we could maybe find a way to make Flask read in the page breaks and add a new paragraph for each one